swagger.yaml

```diff
@@ -107,8 +107,6 @@ paths:
           description: "Provides a hierarchical list of navigation items with their links, copy and localisation references"
           schema:
             $ref: '#/definitions/Navigation'
-        401:
```

Comment:
You can technically still get a `401` if you try to access the `/navigation` endpoint when `cfg.EnablePrivateEndpoints` is set to `true`

So I wouldn't suggest removing this or the logic needs to change so that 401 does not occur

Reply:
So you are referring to configuration value which ultimately changes how the application works and in two different configurations results in swagger spec being different, this is all slightly backwards but Im not doing a refactor here.

Basically what should happen here is that all authentication should cease to exist in the app, as none of the endpoints need to be protected by authentication and hence this app doesn't need to have a configuration to run it in web or publishing mode. This would only take place when we have endpoints that are writing to the database.

Reply:
Got you. That makes complete sense